Scripts/SQLServer/DataTrimmingScripts/Delete_Insights_Data_DBO.sql

```diff
@@ -18,6 +18,11 @@ CREATE OR ALTER PROCEDURE [dbo].[Delete_Insights_Data_DBO]
 	@BatchSize INT = 100000
 AS
 BEGIN
+	IF IS_SRVROLEMEMBER('sysadmin') != 1
```

Comment:
Do we know if people are comfortable using sysadmin for Insights scripts? @sysm do you know what roles other customers scripts required?